lib/functions-custom.php

```diff
@@ -55,3 +55,42 @@ function orderby_meta_date( $query ) {
   }
 }
 add_action( 'pre_get_posts', 'orderby_meta_date' );
+
+// get_posts for actividades related with event
+function get_event_actividades($event_id) {
+  $args = array (
+  'post_type'              => array( 'actividad' ),
+  'posts_per_page'         => '-1',
+  'meta_query'             => array(
+      array(
+        'key'       => '_igv_related_event',
+        'value'     => $event_id,
+      ),
+    ),
+  );
+
+  return get_posts($args);
+}
+
+// return order number of actividad related with event
+function get_actividad_num($actividad_id) {
```

Comment:
yh this works but I think it should be hooked to `save_posts` for this post type and save the value in a metafield rather than doing it each time for display. You can probably use this function in the hook still tho